### PR TITLE
overwrite the validation in order to change the errorString set

### PR DESCRIPTION
**Pull request by yij793**
address: https://nanopay.atlassian.net/browse/CPF-3728
note: currently the siginUp  view job title section error message is `Please enter a Your Job Title`. I added 
validationPredicates for this section in order to change the errorString to `Please select a Job Title`.  please let me know if there is a better way to do this.

### CODE REVIEW

File: src/foam/nanos/u2/navigation/SignUp.js [hunk 1]
```diff
@@ -95,6 +95,15 @@ foam.CLASS({
         class: 'foam.u2.TextField',
         placeholder: 'Staff accountant'
       },
+      validationPredicates: [
+        {
+          args: ['department'],
+          predicateFactory: function(e) {
+            return e.NEQ(foam.nanos.u2.navigation.SignUp.DEPARTMENT, '');
+          },
+          errorString: 'Please select a Job Title'
```
Comment (jlhughes) [inline src/foam/nanos/u2/navigation/SignUp.js]: Is this a selection/drop-down?  as that is what 'select' suggests, while 'enter' suggest free form entry.

Comment (yij793) [reply src/foam/nanos/u2/navigation/SignUp.js]: this isn't  a selection/drop-down. just an text input that only check if this section is empty or not

Comment (jlhughes) [reply src/foam/nanos/u2/navigation/SignUp.js]: Then 'Please enter a Job Title' would be more appropriate.

Comment (TW80000) [reply src/foam/nanos/u2/navigation/SignUp.js]: ^ I agree about 'enter' instead of 'select', but want to add that "job title" shouldn't be capitalized

Comment (yij793) [reply src/foam/nanos/u2/navigation/SignUp.js]: Ok, chang made

**Review by jlhughes — APPROVED** [review]

**Review by TW80000 — APPROVED** [review]

**Pull request merged**